### PR TITLE
Create script.txt

### PR DESCRIPTION
**Pull request by silbunsa**
Initial folder structure will upload the code shortly for review.

### CODE REVIEW

**Review by zeshengchen — CHANGES_REQUESTED** [review]
Please explain what is content in "TTS_with_bark/script.txt" and why upload this file!

Moreover, it is preferable to include all necessary code as ONE pull request. Such a pull request is NOT really a pull request!

**Comment by silbunsa** [comment]
I created the folder structure for the code. Script.txt is an example text file for translation.

**Review by zeshengchen — CHANGES_REQUESTED** [review]
Should file "TextToSpeech.ipynb" be under root? If so, why?

**Review by zeshengchen — CHANGES_REQUESTED** [review]
For "TTS_bark_mainipynb.ipynb", for the last line of the code, which is commented out. If it is no longer useful, please simply remove it!

**Comment by silbunsa** [comment]
> Should file "TextToSpeech.ipynb" be under root? If so, why?
It was a mistake, it shouldn't be there, I am removing the file.

**Comment by silbunsa** [comment]
> For "TTS_bark_mainipynb.ipynb", for the last line of the code, which is commented out. If it is no longer useful, please simply remove it!

I was confused of how the tts.wav file should be used, saved or sent directly. I am commenting out the last line as to save it as for now.

**Review by zeshengchen — APPROVED** [review]

**Review by roh-it — APPROVED** [review]

**Review by SaiSurajMatta — APPROVED** [review]

File: TTS_with_bark/TTS_bark_mainipynb.ipynb
Comment (ChiragBellara) [inline TTS_with_bark/TTS_bark_mainipynb.ipynb:1]: Can you add what each of these parameters implies and what are the possible values for them?
```
semantic_tokens = generate_text_semantic(
        sentence,
        history_prompt=SPEAKER,
        temp=GEN_TEMP,
        min_eos_p=0.05,  # this controls how likely the generation is to end
    )
```

Also, is it possible to make this whole thing a function that takes the name of a text file as input and returns the name and location of the audio file?
```
script_file = "script.txt"
with open(script_file, "r") as file:
  script = file.read()
script.replace("\n", " ").strip()


sentences = nltk.sent_tokenize(script)
SPEAKER = "v2/hi_speaker_6"
silence = np.zeros(int(0.25 * SAMPLE_RATE))  # quarter second of silence

GEN_TEMP = 0.6
SPEAKER = "v2/hi_speaker_3"
silence = np.zeros(int(0.25 * SAMPLE_RATE))  # quarter second of silence

pieces = []
for sentence in sentences:
    semantic_tokens = generate_text_semantic(
        sentence,
        history_prompt=SPEAKER,
        temp=GEN_TEMP,
        min_eos_p=0.05,  # this controls how likely the generation is to end
    )

    audio_array = semantic_to_waveform(semantic_tokens, history_prompt=SPEAKER,)
    pieces += [audio_array, silence.copy()]

Audio(np.concatenate(pieces), rate=SAMPLE_RATE)
scipy.io.wavfile.write("text_speech.wav", rate=SAMPLE_RATE, data=audio_array)
```

Note: I am unable to comment on the file directly as the file is too big to load on the website.

**Review by ChiragBellara — CHANGES_REQUESTED** [review]

File: TTS_with_bark/TTS_bark_mainipynb.ipynb
Comment (silbunsa) [reply TTS_with_bark/TTS_bark_mainipynb.ipynb:1]: Changed it as you suggested, Check it!

**Review by ChiragBellara — APPROVED** [review]
Thanks. This is good to merge.

**Pull request merged**